src/lava/utils/plots.py

```diff
@@ -0,0 +1,45 @@
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+# See: https://spdx.org/licenses/
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+
+def raster_plot(spks, stride=6, fig=None, color='b', alpha=1):
+    """Generate raster plot of spiking activity.
+
+    Parameters
+    ----------
+    spks : np.ndarray shape (num_neurons, num_timesteps)
```

Comment:
Could you please remove everything after `np.ndarray` here. Instead, please incorporate a full but concise description of the expected format of the `spikes` parameter in the description (next line).